doc/content/gateways/lora-basics-station/lns.md

```diff
@@ -29,16 +29,16 @@ On your gateway, set the following configuration fields.
 
 The `<server-address>` is the address of {{% tts %}}. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.
 
-The `<optional-gateway-api-key>` is the API Key you created above, if using Token Authentication. Copy your gateway API Key in to a `gateway-api.key` file (the filename is not important) as an HTTP header in the following format:
+The `<gateway-api-key>` is the API Key you created above. Create a file named `gateway-api.key` and copy your gateway API Key in as an HTTP header in the following format:
 
 ```
-Authorization: <optional-gateway-api-key>
+Authorization: <gateway-api-key>
```

Comment:
Aren't we mandating that this starts with `Bearer`? How does this work?

Reply:
cc @KrishnaIyer

Reply:
We use `Bearer` only for the CUPS token so I've added a commit to fix that.

Reply:
Wait, we don't accept `Bearer <api-key>` in Basic Station LNS? Why is there a difference?

Reply:
No we accept `Bearer <api-key>`, but we don't use it for some historic reason that I've forgotten. We can make the two consistent.

Reply:
Then I would suggest to document it including `Bearer`, as that's the most correct way to do it.